packages/itwinui-css/src/select/select.scss

```diff
@@ -102,11 +102,16 @@
   align-items: center;
   block-size: 80%;
   max-block-size: calc(var(--iui-size-s) * 3);
+
+  // Truncate when only one tag is present
+  &:only-child {
+    max-inline-size: 100%;
```

Comment:
now i'm wondering if we even need the `:only-child`. shouldn't the max-width always be 100%, regardless of whether it's the only child?

Reply:
Ohh, that's true, removed `:only-child` (https://github.com/iTwin/iTwinUI/pull/1716/commits/877b8b0df93811001f5c8a41e680b338bb07ca22)